bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/ConfigurationImpl.java

```diff
@@ -49,22 +43,22 @@ protected void init() {
 
         resourceResolver = request.getResourceResolver();
 
-        pageManager = resourceResolver.adaptTo(PageManager.class);
-        currentPage = pageManager.getContainingPage(request.getResource());
+        PageManager pageManager = resourceResolver.adaptTo(PageManager.class);
 
-        Page page = currentPage;
+        Page page = pageManager.getContainingPage(request.getResource());
 
         while (page != null) {
             final String confPath = page.getProperties().get("cq:conf", String.class);
 
             if (StringUtils.isNotBlank(confPath)) {
+                rootPage = page;
```

Comment:
This should be th first page that has a pwa conf? His will be a page w a cq:conf but that may not had a pwa conf?

Reply:
True and that's all we need. Once we have a rootPage defined, we know this site supports pwa and in the footer scripts we can make a call to get the rootPage beneath which the SW could be loaded

Reply:
Should we be specifying the scope on the Config or letting it naturally resolve based on which node has the cq:conf. What is the specified path does not match the actual path?

Reply:
I have added a change to have the scope to the configuration